acme_diags/driver/utils/dataset.py

```diff
@@ -156,6 +156,20 @@ def get_climo_variable(self, var, season, extra_vars=[], *args, **kwargs):
         return variables[0] if len(variables) == 1 else variables
 
 
+    def get_static_variable(self, static_var, primary_var):
+        # TODO: is this function coded correctly? Need to do anything to deal with different resolutions/grids?
+        #  We don't have to address every use case now -- just what we need for streamflow diags
+        if self.ref:
+            # Get the reference variable from timeseries files.
+            data_path = self.parameters.reference_data_path
+        elif self.test:
+            # Get the test variable from timeseries files.
+            data_path = self.parameters.test_data_path
+        file_path = self._get_timeseries_file_path(primary_var, data_path)
+        f = cdms2.open(file_path)
```

Comment:
with cdms2.open(file_path) as f:
        return f[static_var]

Reply:
I don't think this will work. I keep getting `cdms2.error.CDMSError: Cannot read from closed file`. It seems like `cdms2.open` requires that file to stay open.

Reply:
i think it should be `f(static_var)`. if `with` statement won't work, we can use cdms2.open directly, but close the file afterwards.